frontend/src/component/NavBar.jsx

```
@@ -0,0 +1,63 @@
+        {/* Desktop Menu */}
+        <div className="hidden md:flex items-center gap-6">
+          <Link to="/login">
+            <button className="text-white px-4 py-2 border border-white rounded-md hover:bg-white hover:text-black transition">Login</button>
+          </Link>
+          <Link to="/signup">
+            <button className="text-black px-4 py-2 bg-white rounded-md hover:bg-gray-300 transition">Signup</button>
+          </Link>
+        </div>
+
+        {/* Menu Button for Mobile */}
+        <button onClick={() => setIsOpen(true)} className="text-white md:hidden">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)